src/MySTMarkdownCell.tsx

```diff
@@ -56,19 +94,30 @@ export class MySTMarkdownCell
     const { references, frontmatter } = notebook.myst;
 
     const children = useParse(mdast as any, renderers);
-
+    const metadata = this.model.metadata.get(
+      metadataSection
+    ) as IUserExpressionMetadata[];
     render(
       <ThemeProvider
         theme={Theme.light}
         Link={linkFactory(notebook)}
         renderers={renderers}
       >
-        <TabStateProvider>
-          <ReferencesProvider references={references} frontmatter={frontmatter}>
-            {isFirstCell && <FrontmatterBlock frontmatter={frontmatter} />}
-            {children}
-          </ReferencesProvider>
-        </TabStateProvider>
+        <CellMetadataProvider
+          metadata={metadata}
+          trusted={this.model.trusted}
+          rendermime={this.__rendermime}
+        >
```

Comment:
We add a cell metadata provider to give access to these values in the inline renderers.